### ISSUE TITLE
lift limitation of 10000 tracks in a playlist

### ISSUE THREAD

**Issue by lilmike** [mopidy/mopidy#1600]
I had asked on the IRC channel a while back if there could be support for more than 10000 songs in a playlist, and was told it was to provide compatibility with mpd clients that can't handle more than that. Apparently there is some kind of MPD extension (though I can't seem to find where the details of it reside right now) that allows for that, and it's been implemented for a while. I was just opening this issue to get it in an official feature request form and to get more discussion on it.
-Michael.

**Comment by kingosticks** [comment]
There's a core option called `max_tracklist_length` that controls this limit. What's wrong with you just setting this higher?  https://docs.mopidy.com/en/latest/config/#confval-core/max_tracklist_length

**Comment by adamcik** [comment]
The settings validator has `max=10000` set. So someone just needs to remove that and update the docs.

**Comment by lilmike** [comment]
Hi,
I'm pretty sure I tried that, and I got a trace back about Max tracklist length being only allowed to be 10,000 or less.
Michael.

Sent from my iPhone

On Feb 11, 2017, at 3:23 PM, Nick Steel <notifications@github.com<mailto:notifications@github.com>> wrote:


There's a core option called max_tracklist_length that controls this limit. What's wrong with you just setting this higher? https://docs.mopidy.com/en/latest/config/#confval-core/max_tracklist_length
